documentation/typescript/adding-custom-login-form-with-spring-security.asciidoc

```diff
@@ -0,0 +1,176 @@
+---
+title: Adding a Custom Login Form with Spring Security
+order: 91
+layout: page
+---
+
+ifdef::env-github[:outfilesuffix: .asciidoc]
+
+= Adding a Custom Login Form with Spring Security
+
+Instead of using the <<adding-login-form-with-spring-security#, default Spring login form>>, you may want to use your own customized form. This article describes how to do that.
+
+== Dependencies and Server Configuration
+
+The dependencies and server configuration is the same as described in <<adding-login-form-with-spring-security#, Adding a Login Form with Spring Security>>, with a difference that you need to specify the URL of the custom login view.
+
+.SecurityConfig.java
+[source,java]
+----
+@EnableWebSecurity
+@Configuration
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+  @Override
+  protected void configure(HttpSecurity http) throws Exception {
+    // Vaadin already handles csrf.
+    http.csrf().disable();
```

Comment:
Should we do some specific ignore instead of disable, perhaps?

https://github.com/vaadin/flow/issues/9202

Reply:
done, updated to 
```
// Ignore the login processing url and vaadin endpoint calls
http.csrf().ignoringAntMatchers("/login", "/connect/**");
```